src/components/cluster/new/cluster_creation_duration.js

```diff
@@ -0,0 +1,29 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+class ClusterCreationDuration extends React.Component {
+  render() {
+    let minutes = 0;
+
+    if (this.props.stats.p75) {
```

Comment:
what does `p75` stands for?

Reply:
The `p75` attribute is the  75th percentile of a value distribution. See https://docs.giantswarm.io/api/#operation/getInfo

![image](https://user-images.githubusercontent.com/273727/58934302-58f14200-876a-11e9-881c-b106cf0299ff.png)